Assignment2/bingyu-g-BingyuGuan.ttl

```diff
@@ -0,0 +1,9 @@
+@base <http://www.oeg-upm.net>
+@prefix xsd: <http://www.oeg-upm.net/XMLSchema#>
+:Class01	:includes :Sensor029;
+   	:includes :Computer101.	
+:Sensor029 :hasMeasurement :Measurement8401.
+:Measurement8401 :hasTemperature  �29�^^xsd:integer;
+		         :atTime �2010-06-12T12:00:12�^^xsd:datetime.
+:Computer101  :hasOwner  :User10A.
+:User10A   :hasName  :�Pedro�.
```

Comment:
Wrong syntax here.
Apart from that, not too important, but for future submissions, when you use a word editor instead of a text/code editor, it may change things like the " symbol, which is not the right one. Be careful when you do this with code.